app/lib/mammoth/channels.rb

```diff
@@ -3,9 +3,41 @@ module Mammoth
   class Channels
     class NotFound < StandardError; end
 
+    GO_BACK = 24 # number of hours back to fetch statuses
     ACCOUNT_RELAY_AUTH = "Bearer #{ENV.fetch('ACCOUNT_RELAY_KEY')}"
```

Comment:
originally I thought, 2 hours should be good to run the cron with overlap. But I forgot about past posts where engagement goes up over time. If the post was older than a couple hours and got a lot of engagement, it wouldn't have gotten picked up. So we want to reach back 24 hours, _and then_ filter for engagement